src/oogeso/core/optimiser.py

```diff
@@ -351,7 +354,8 @@ def _specify_constraints(self):
         else:
             logger.debug("No emission_intensity_max limit specified")
         # 4.3 electrical reserve margin:
-        el_parameters = self.all_networks["el"].carrier_data
+        # Fixme: This is prone to errors, and depends on the class El(Network).__name__ not changing.
+        el_parameters: dto.CarrierElData = self.all_networks["el"].carrier_data  # Fixme: Static type error.
```

Comment:
And this

Reply:
Need to change if we rename El(Network)? See if we can find a better solution than this dictionary.